app-web/__tests__/pages/Index.test.js

```diff
@@ -89,7 +89,7 @@ describe('Home Page', () => {
     expect(Alert).toBeInTheDocument();
   });
 
-  test('when there is an empty search the alert box does not show and all results show instead', () => {
+  test('when there is an empty search the alert box does not show and neither do cards', () => {
```

Comment:
so by empty search do we mean that the path is `developer.gov.bc.ca/?q=` where a user search with an empty string?

Reply:
Yeah exactly

Reply:
I think we should reword this,  (this is initially my bad).
Something Like... when a blank search is entered, only the topics are visible

Reply:
something like that? 
We can then assert that card don't show
alerts don't show
and topics/collections do show.

Reply:
Good call, how about "When a blank search is entered, we can assert that cards and alerts don't show but topics/collections do"?

Reply:
sure but ditch the 'we can assert' part. the 'expect' function is the assertion

Reply:
gotcha, consider it done

Reply:
i should have bulletted my example from above:P

> something like that?
> We can then assert that card don't show
> alerts don't show
> and topics/collections do show.
> 
> what do you think?

Reply:
haha no worries, all good